vg/len.go

```diff
@@ -4,37 +4,24 @@
 
 package vg
 
-import (
-	"strconv"
-	"strings"
-)
+import "gonum.org/v1/plot/font"
 
 // A Length is a unit-independent representation of length.
 // Internally, the length is stored in postscript points.
-type Length float64
+type Length = font.Length
```

Comment:
I'm not sure about this direction. I can see philosophical arguments in both directions though. Is the import in the other direction possible?

Reply:
I believe that's the only possible import order (if we keep the 2 packages separate).

consider:
```go
package vg

type Canvas interface {
    ...
    FillString(fnt Font, pt Point, txt string)
}

type Font struct {
    Size Length
    ...
}
```

`Font` needs to know about the `Length` type.
`Canvas` needs to know about the `Font` type.

so you can't have `Length` together with `Canvas` and keep `Font` in a different package.
I believe it needs to be:
- a) `(Canvas) -> (Font) -> (Length)`, or
- b) `(Canvas) -> (Font, Length)`, or
- c) `(Canvas, Font, Length)`

for aesthetical reasons, I chose `b)` (and in another thread you seemed to indicate you'd be -like I was- in favor of having a dedicated `font` package)

happy to go either way, though.

Reply:
Yeah. I'm torn. I think it's reasonable the way you have it given that points derive their existence from fonts. I would be good to have someone else's input as well though just to ensure it's not shared delusion.